### PR TITLE
add errors package to wpwithin

### PR DESCRIPTION
**Pull request by wberendt**

### CODE REVIEW

File: wpwithin/errors/errors.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+type ErrorId int
+
+const (
+	UNKNOWN = 0 + iota
```
Comment (BartMoroz) [inline wpwithin/errors/errors.go]: why 0? iota is set to 0 after each 'const' keyword

File: wpwithin/errors/errors.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	NO_DATA
```
Comment (BartMoroz) [inline wpwithin/errors/errors.go]: either use the ErrorId type here, or stick to int

File: wpwithin/errors/errors.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+var errors = [...]WpwError{
+	UNKNOWN: {"UNKNOWN", GENERAL, "unknow error"},
+	// add new error below
+	NO_DATA: {"NO_DATA", DATA, "lack of data"},
+}
+
+// Error return formated error for specified error (e).
+func (e WpwError) Error() error {
+	t := time.Now()
```
Comment (BartMoroz) [inline wpwithin/errors/errors.go]: we dont need time, as our logger adds it

File: wpwithin/errors/errors.go [hunk 4]
```diff
@@ -0,0 +1,50 @@
+package errors
+
+import (
+	"fmt"
+	"strings"
+	"time"
```
Comment (BartMoroz) [inline wpwithin/errors/errors.go]: time not needed

File: wpwithin/errors/errors.go (hunk 1 above)
Comment (wberendt) [reply wpwithin/errors/errors.go]: ok

File: wpwithin/errors/errors.go (hunk 2 above)
Comment (wberendt) [reply wpwithin/errors/errors.go]: ok

File: wpwithin/errors/errors.go (hunk 3 above)
Comment (wberendt) [reply wpwithin/errors/errors.go]: you right, ok

File: wpwithin/errors/errors.go [hunk 5]
```diff
@@ -3,14 +3,14 @@ package errors
...
 type ErrorId int
 
 const (
-	UNKNOWN = 0 + iota
+	UNKNOWN ErrorId = iota
 	NO_DATA
+	DIVISION_BY_ZERO
 )
 
 type ErrorType int
```
Comment (BartMoroz) [inline wpwithin/errors/errors.go]: Do we need this?

**Pull request merged**